python_modules/dagster-graphql/dagster_graphql/schema/instigation.py

```diff
@@ -373,6 +373,7 @@ def resolve_evaluationResult(self, graphene_info: ResolveInfo):
                     last_tick_completion_time=None,
                     last_run_key=None,
                     last_sensor_start_time=None,
+                    log_key=None,
```

Comment:
Will need to do some work to get these logs in for ad-hoc evaluation ticks from the UI. Probably will create an issue for this.

Reply:
This whole scheme needs a bit of rework if you're game I think - particularly in cloud, it needs to become more async i think and actually execute on the daemon, or it runs into nginx timeouts long before the sensor tick actually finishes (will become more acute once sensor ticks can be longer)